pal_tiago/tiago.xml

```diff
@@ -0,0 +1,222 @@
+    <mesh name="wheel" file="wheels/wheel.stl"/>
+    <mesh name="caster_1" file="wheels/caster_1.stl"/>
+    <mesh name="caster_2" file="wheels/caster_2.stl"/>
+    <mesh name="torso_fix" file="torso/torso_fix.stl"/>
+    <mesh name="torso_lift_with_arm" file="torso/torso_lift_with_arm.stl"/>
+    <mesh name="head_1" file="head/head_1.stl"/>
+    <mesh name="head_2" file="head/head_2.stl"/>
+    <mesh name="arm_1" file="arm/arm_1.stl"/>
+    <mesh name="arm_2" file="arm/arm_2.stl"/>
+    <mesh name="arm_3" file="arm/arm_3.stl"/>
+    <mesh name="arm_4" file="arm/arm_4.stl"/>
+    <mesh name="arm_5-wrist-2017" file="arm/arm_5-wrist-2017.stl"/>
+    <mesh name="arm_6-wrist-2017" file="arm/arm_6-wrist-2017.stl"/>
+    <mesh name="gripper_link" file="gripper_link.stl"/>
+    <mesh name="gripper_finger_link" file="gripper_finger_link.stl" scale="-1 -1 1"/>
+    <mesh name="gripper_finger_link1" file="gripper_finger_link.stl"/>
+  </asset>
+
+  <default class="main">
+    <geom contype="0" conaffinity="1" group="1"/>
+    <default class="arm">
+      <geom contype="1" conaffinity="1"/>
+    </default>
+  </default>
+
+  <worldbody>
+    <body name="base_link" pos="0 0 0">
+      <joint name="reference" type="free"/>
+      <geom pos="0 0 0.0985" type="mesh" density="0" rgba="1 1 1 1" mesh="base"/>
+      <geom pos="0.202 0 0.0945" type="mesh" density="0" rgba="0.1 0.1 0.1 1" mesh="sick_tim551"/>
+      <geom pos="-0.183 0.183 0.2715" quat="0.382683 0 0 0.92388" type="mesh" density="0" rgba="0.1 0.1 0.1 1"
+        mesh="srf05"/>
+      <geom pos="-0.259 0 0.2715" quat="0 0 0 1" type="mesh" density="0" rgba="0.1 0.1 0.1 1" mesh="srf05"/>
+      <geom pos="-0.183 -0.183 0.2715" quat="-0.382683 0 0 0.92388" type="mesh" density="0" rgba="0.1 0.1 0.1 1"
+        mesh="srf05"/>
+      <geom pos="-0.201 0.1062 0.2935" type="mesh" density="0" rgba="0 0 0 1" mesh="antenna"/>
+      <geom pos="-0.201 -0.1062 0.2935" type="mesh" density="0" rgba="0 0 0 1" mesh="antenna"/>
+      <geom pos="-0.062 0 0.2915" type="mesh" density="0" rgba="0.9 0.9 0.9 1" mesh="torso_fix"/>
+      <geom size="0.09 0.1 0.275" pos="-0.062 0 0.5685" type="box" density="0" rgba="0.1 0.1 0.1 1"/>
+      <geom pos="0 0 0.13" type="mesh" density="0" rgba="1 0.487 0 1" mesh="base_ring"/>
+      <!-- base link inertial manually added from urdf -->
+      <!-- <mass value="28.26649"/>
+      <inertia ixx="0.465408937" ixy="0.002160024" ixz="-0.001760255" iyy="0.483193291" iyz="-0.000655952"
+      izz="0.550939703"/> -->
+      <inertial pos="0 0 0" mass="28.26649"
+        fullinertia="0.465408937 0.483193291 0.550939703 0.002160024 -0.001760255 -0.000655952"/>
+      <body name="suspension_right_link" pos="0 0 0.0985">
+        <inertial pos="0 0 -0.02" mass="10" diaginertia="1 1 1"/>
+        <joint name="suspension_right_joint" pos="0 0 0" axis="0 0 1" type="slide" range="-0.005 0.005" damping="100"/>
+        <body name="wheel_right_link" pos="0 -0.2022 0" quat="0.707107 -0.707107 0 0">
```

Comment:
MuJoCo will normalize quaternions for you. Therefore, for quats like `.7 -0.7 0 0`, you can more cleanly write it as `1 -1 0 0`.

Reply:
Done